src/commands/workspace/create.ts

```diff
@@ -0,0 +1,110 @@
+/*********************************************************************
+ * Copyright (c) 2019-2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { boolean, string } from '@oclif/parser/lib/flags'
+import { cli } from 'cli-ux'
+import * as Listr from 'listr'
+import * as notifier from 'node-notifier'
+
+import { CheHelper } from '../../api/che'
+import { accessToken, cheNamespace, listrRenderer } from '../../common-flags'
+export default class Create extends Command {
+  static description = 'Creates a workspace from devfile'
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    chenamespace: cheNamespace,
+    devfile: string({
+      char: 'f',
+      description: 'path or URL to a valid devfile',
+      env: 'DEVFILE_PATH',
+      required: true,
```

Comment:
We can make it optional. If flag isn't set then let's use the devfile from the current directory

Reply:
ok